api/tests/integ/test_integration.py

```diff
@@ -773,6 +773,34 @@ def test_list_measurements_ZZ(client):
     assert resp.status_code == 403
 
 
+def test_list_measurements_filter_software_version_nomatch(client):
+    # https://github.com/ooni/backend/issues/15
+    url = "measurements?since=2021-07-09&until=2021-07-10&software_version=9.9.9"
```

Comment:
Because the functionality is advertised as a comma separated list, can all the tests here be using the comma separated version of the parameter rather than using just a scalar? 🙏